plugin/src/templates/handlers.ts

```diff
@@ -0,0 +1,151 @@
+import type { Handler, HandlerEvent } from '@netlify/functions'
+import { stripIndent as javascript } from 'common-tags'
+import type { GatsbyFunctionRequest } from 'gatsby'
+import type {
+  getData as getDataType,
+  renderHTML as renderHTMLType,
+  renderPageData as renderPageDataType,
+} from 'gatsby/cache-dir/page-ssr'
+import type { IGatsbyPage } from 'gatsby/cache-dir/query-engine'
+
+// These are "require()"d rather than improted so the symbol names are munged,
```

Comment:
TIL that munged exists as a word

![image](https://user-images.githubusercontent.com/11183523/150743418-2799d993-6795-4dfe-8c11-80843a442b5c.png)